invenio_communities/views/decorators.py

```diff
@@ -69,4 +76,21 @@ def view(**kwargs):
             raise PermissionDeniedError()
         kwargs['endpoint'] = endpoint
         return f(**kwargs)
+
+    return view
+
+
+def pass_invitation(f):
+    """Decorate to retrieve the community-invitation using the requests service.
```

Comment:
copied docstring

Reply:
not the right description? :sweat_smile: